tests/test_xml/generated_xml/CXR333_IM-1594-1001.json

```diff
@@ -0,0 +1,119 @@
+{
+  "py/object": "forte.data.data_pack.DataPack",
+  "py/state": {
+    "_creation_records": {
+      "textdata_preprocessor.FindingsExtractor": {
+        "py/set": [
+          1
+        ]
+      },
+      "textdata_preprocessor.ImpressionExtractor": {
+        "py/set": [
+          2
+        ]
+      },
+      "textdata_preprocessor.ParentImageExtractor": {
+        "py/set": [
+          3
+        ]
+      }
+    },
+    "_field_records": {
+      "textdata_preprocessor.FindingsExtractor": {
+        "py/set": [
+          {
+            "py/tuple": [
+              1,
+              "has_content"
+            ]
+          }
+        ]
+      },
+      "textdata_preprocessor.ImpressionExtractor": {
+        "py/set": [
+          {
+            "py/tuple": [
+              2,
+              "has_content"
+            ]
+          }
+        ]
+      },
+      "textdata_preprocessor.ParentImageExtractor": {
+        "py/set": [
+          {
+            "py/tuple": [
+              3,
+              "has_content"
+            ]
+          }
+        ]
+      }
+    },
+    "links": [],
+    "groups": [],
+    "_meta": {
+      "py/object": "forte.data.data_pack.Meta",
+      "py/state": {
+        "pack_name": "CXR333_IM-1594-1001",
+        "_pack_id": 25960070430737649482897247911429098342,
+        "language": "eng",
+        "span_unit": "character"
+      }
+    },
+    "_text": "PARENTIMAGE /home/jiachen.li/iu_xray_images/CXR333_IM-1594-1001.png FINDINGS the lungs and pleural spaces show no acute abnormality. lungs are mildly hyperexpanded. heart size and pulmonary vascularity within normal limits. IMPRESSION no acute pulmonary abnormality.",
```

Comment:
Looks like you have put your local path in the data. I think we don't need this since we have the `FilePath` object capturing the path.

Reply:
Cool, I will remove the ``PARENTIMAGE ``object

Reply:
The travis errors have been fixed.